adapter/src/components/ErrorBoundary.js

```diff
@@ -61,8 +61,14 @@ export class ErrorBoundary extends Component {
         })
     }
 
+    handleSafeLoginRedirect = () => {
+        window.location.href =
+            this.props.url + '/dhis-web-commons/security/login.action'
```

Comment:
I think this prop would be btter named `baseUrl`? and maybe be more defensive with the url construction here, in case the base URL has a trailing slash, to avoid a double slash